### ISSUE TITLE
How to train with other depth annotations

### ISSUE THREAD

**Issue by ruili3** [Brummi/MonoRec#41]
Hi,

I'm following your work and wish to train the model with the dense annotated depth. Can I simply set `lidar_depth=true` and `dso_depth=false` in monorec_depth.json file?

Moreover, in the .json file, the batchsize=8 and n_gpu=8, so is there totally 8*8=64 batches during the distributed learning? Since I do not have so many cards, so I cant train the model with 64 total batches a time. With your experience, should I set the learning rate with a higher value or try to train with more epochs than 70? Thank you a lot!

**Comment by Brummi** [comment]
Hi,
in theory, it should basically work out of the box with dense depths, except for the correct loading of the files. (lidar depth often has a weird format).

No, the batch size is total combined over all cards. That means, in this configuration it would be 1 sample per GPU. But the script will take _up to_ 8 GPUs, if they are available. I mostly trained with one or two GPUs.

Best,
Felix
